test/unit/bootstrap.js

```diff
@@ -17,7 +17,10 @@ global.URL = function URL(url) {
     };
 };
 
-global.Event = () => {};
+// https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#sec-instanceofoperator
+// ES standard requires that left-hand side shall have a prototype method, this
+// is not the case for lambdas.
```

Comment:
so you should update `global.requestAnimationFrame` too?

Reply:
This issue arises only for check that some function (here `Event`) is an `instanceof` of some types (here `Event`).
There is no such use of this operator for `requestAnimationFrame`.

Reply:
ok :+1: